rayon-core/src/job.rs

```diff
@@ -64,24 +64,24 @@ impl JobRef {
 /// executes it need not free any heap data, the cleanup occurs when
 /// the stack frame is later popped.  The function parameter indicates
 /// `true` if the job was stolen -- executed on a different thread.
-pub(super) struct StackJob<L, F, R>
+pub(super) struct StackJob<'a, L: 'a, F, R>
```

Comment:
instead of adding `'a`, can we implement `Latch` for `&'a mut L` where `L: Latch`?

Reply:
or, probably,, just `&L`

Reply:
(Why are we using `&mut L` in the first place?)

Reply:
This should be addressed now.